### DISCUSSION TITLE
Grolsch beer has its origins where?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#575] [Galaxy Trivia] [answered]
**Bonus:** What are two other famous beers from the same country?

**Comment by peterbaccaro** [comment, marked as answer]
Grolsch beer has its origins in the **city of Enschede**, which is located in the **eastern part of the Netherlands**. The brewery was founded in 1615 by Willem Neerfeldt in the city of Groenlo, which was then part of the Dutch Republic. The name "Grolsch" is derived from the name of the city of Groenlo, which was also known as Grol in the past. Grolsch is known for its distinctive swing-top bottle, which was introduced in the 1890s and has become an iconic feature of the brand. The brewery is still in operation today and produces a range of different beers, including its flagship lager, Grolsch Premium Pilsner.

Two other famous beers from the Netherlands are:

**Heineken** - This is perhaps the most well-known Dutch beer around the world, a lager that was first brewed in Amsterdam in 1873.

**Amstel** - Another popular Dutch beer, Amstel is a light lager that was first brewed in Amsterdam in 1870.

**Reply by ParanoidUser** [reply]
@peterbaccaro, and what is your beer of choice? 🍺

**Reply by peterbaccaro** [reply]
**Peroni** is sometimes my beer of choice or sometimes a pale ale such as **Camden Pale Ale** or **Brew Dog IPA**.

**Reply by peterbaccaro** [reply]
Do you have a favourite beer?

**Reply by ParanoidUser** [reply]
Absolutely-doodle-doo! As we're stepping into the spring/summer season, my fav so far:
 - 🥇 [Samuel Smith's Pure Lager](https://samuelsmithsbrewery.co.uk/shop/bottles/lagers/pure-brewed-organic-lager)
 - 🥈 [Tucher's Helles Hefe Weizen](http://www.tucher.de/unsere-biere/unser-sortiment/weizenbiere)
 - 🥉 [Kona's Big Wave Golden Ale](https://konabrewingco.com/our-beers)

**Reply by ParanoidUser** [reply]
> **Peroni** is sometimes my beer of choice or sometimes a pale ale such as **Camden Pale Ale** or **Brew Dog IPA**.

Your list is much appreciated! Excellent choices. I've tried Peroni and Bew Dog, but **Camden Pale Ale** now reside in my watch list. Thanks for the recommendation! Cheers! 🍻
